clients/web/src/templates/widgets/editCollectionWidget.pug

```diff
@@ -4,10 +4,10 @@
       .modal-header
         button.close(data-dismiss="modal", aria-hidden="true", type="button") &times;
         h4.modal-title
-          if (collection)
-            | Edit collection
-          else
+          if (create)
```

Comment:
I'm not clear what you mean by compatibility. This is an implementation detail of a template that's already tightly bound to being used by `EditCollectionWidget`.

FYI, the `EditCollectionWidget.create` instance variable existed before, it was just not being injected into the template.